dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.41/V2_41_32__Add_created_lastupdated_column_in_customicon.sql

```diff
@@ -0,0 +1,3 @@
+alter table customicon add column if not exists "created" timestamp;
+alter table customicon add column if not exists "lastupdated" timestamp;
+alter table customicon add column if not exists "custom" BOOLEAN;
```

Comment:
Do we really need this information on the DB?
This value will always be true, there is no use case where this will be false.